templates/zerver/integrations/git.md

```diff
@@ -1,31 +1,41 @@
+# Zulip Git integration
+
 Get Zulip notifications for your Git repositories!
 
+{start_tabs}
+
 1. {!create-an-incoming-webhook.md!}
 
 1. {!download-python-bindings.md!}
 
-1. {!create-channel.md!}
-
 1. {!change-zulip-config-file.md!}
 
-    You may also need to change the value of `STREAM_NAME`.
+    To customize further, refer to the
+    [configuration options](#configuration-options).
 
-    You can specify the branches that will be used for notifications by modifying
-    the `commit_notice_destination` function. By default,
-    pushes to the `main`, `master`, and `test-post-receive` branches will result in a
-    notification.
+1. Symlink the `post-receive` hook of your Git repository by running:
 
-1. Symlink `/usr/local/share/zulip/integrations/git/zulip_git_config.py`
```

Comment:
This instruction was previously clubbed together as "symlink both".
Alya recommended writing out the instructions, and I was trying to format the instructions for both Windows and Linux/Mac OSes, when I went back to digging into the script and noticed that the configuration file is the same as in other integrations, and the symlink is only for the post-receive git hook.

So, removed that other instruction for symlinking the config file.
And used the Linux command for symlinking the hook to the script, since I noticed that we typically don't mention Windows commands in our integration docs.